tools/src/gen-params.rs

```diff
@@ -95,30 +115,38 @@ fn write_hashes(params_root_dir: &str) {
     }
 }
 
-/// Main entry point for the parameter generation tool
 #[tokio::main]
+/// Main entry point for the parameter generation tool
 async fn main() {
+    env_logger::init();
```

Comment:
If we use a logger, then everything output should be a `trace/info/debug/warn/error`, otherwise everything should be a `println` – but please let us avoid mixing both.